modules/sno-about-sno-cluster-worker-nodes.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: CONCEPT
+[id="sno-about-sno-cluster-worker-nodes_{context}"]
+= About worker nodes on single-node OpenShift clusters
+
+Worker nodes for single-node OpenShift clusters provide additional resources for your cluster. However, because there is only a single control plane node, ingress traffic is, by default, routed through the control plane node. One of the benefits of installing single-node clusters is the simplicity of not having to deal with load-balancing and virtual IPs. Not having a load balancer installed by default increases cost savings and simplifies cluster deployment and configuration.
```

Comment:
>However, because there is only a single control plane node, ingress traffic is, by default, routed through the control plane node.

This is not the reason we route traffic through the single node. The real reason is because before there were workers, there was only that node, and so all DNS entries / traffic was trivially pointed at it. And so we don't want users to have to change that once they add workers, so we by default keep routing all ingress traffic through the single node as we did originally.

I suggest something along the lines of:

```suggestion
Worker nodes for single-node OpenShift clusters provide additional resources for your cluster. Note that unlike multi-node clusters, by default all ingress traffic should still be routed to the single control-plane node, even after adding worker nodes.
```